### PR TITLE
#169039872 fix backend social login and email verification

### PR DESCRIPTION
**Pull request by salviosage**
#### What does this PR do?

- fixing some backend issues 

#### Description of Task to be completed?

- make social login redirect after authenticating someone 

- fixing link sent to email verification to be the frontend link 

- making bio to accept to be set to empty

#### How should this be manually tested?

- cloning this repo login with social login it will redirect you back to the frontend with the token 

- signup with email the verification link is going to be frontend URL

- set your bio back  to null or nothing is going to  work out

#### What are the relevant pivotal tracker stories?
[#169039872](https://www.pivotaltracker.com/story/show/169039872)

### CODE REVIEW

**Review by nkalyesubula — DISMISSED** [review]
LGTM

File: src/controllers/social.js [hunk 1]
```diff
@@ -21,56 +24,49 @@ class Social {
...
-      const newUser = {
-        firstname, lastname, email, username, password
-      };
-      user = await User.addUser(newUser);
-      message = `Account created with password ${pass}, please change your password`;
-      if (data.provider === 'twitter') {
-        message += ' and update your email address';
-        console.log('cccc', message);
+    const type = data.provider;
+    console.log(data);
```
Comment (habinezadalvan) [inline src/controllers/social.js]: remove this console.log

File: src/controllers/user.controller.js [hunk 2]
```diff
@@ -197,10 +201,10 @@ class UserController {
         token
       });
     } catch (error) {
-      const { response: { body: { errors } } } = error;
+      // const { response: { body: { errors } } } = error;
```
Comment (habinezadalvan) [inline src/controllers/user.controller.js]: remove this commented line

**Review by habinezadalvan — COMMENTED** [review]
remove the highlighted lines

**Review by habinezadalvan — APPROVED** [review]

**Review by minega25 — APPROVED** [review]
LGTM

**Pull request merged**